lightcurve/static/lightcurve/js/lightcurve.js

```diff
@@ -74,18 +73,20 @@ $(function() {
     return false;
   });
 
-  $(document).on('click', '.edit-name', function(e) {
-    e.preventDefault();
+  $('.js-edit-name').on('click', function() {
     var filename = $(this).prev(".lightcurve-name").text();
```

Comment:
This shouldn't be named `filename` because it's not a file name - it's just user's input.

Reply:
Done